src/test/resources/org/opa/ideaplugin/lang/parser/fixtures/rules_with_single_predicates.rego

```diff
@@ -9,7 +9,7 @@ withFalseKeyword {
 }
 
 withFunction {
-    isTrue()
+    isTrue
```

Comment:
Was this wrong?

Reply:
yep
> 1 error occurred: policy.rego:5: rego_type_error: undefined function data.play.isTrue

https://play.openpolicyagent.org/p/1jc89W9oXE